src/cpuid.cpp

```diff
@@ -46,7 +46,7 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <cstring> // for std::memset
 #endif
 
-#if TORRENT_HAS_ARM
+#if TORRENT_HAS_ARM && TORRENT_HAS_ARM_NEON
```

Comment:
linaro for raspberry pi. arm-linux-gnueabihf-raspbian
Additionally, this is a linux header, so presumably it should only be used on ``__linux__`` as well, right?